exercise/sitemaps.py

```diff
@@ -18,12 +18,25 @@ def items(self):
             audience=LearningObject.AUDIENCE.COURSE_AUDIENCE,
         )
 
+
+class HiddenBaseExerciseSitemap(AplusSitemap):
+    changefreq = 'daily'
+    base_priority = 0.0
+
+    def items(self):
```

Comment:
The standalone exercise pages for exercises that are included in a chapter were previously not part of the sitemap. However, they still showed up in Google search sometimes. The ``HiddenBaseExerciseSitemap`` with priority of ``0.0`` is an attempt at hiding those pages from Google.

Reply:
From reading about sitemaps, I get the impression that it is not a good idea to include pages in the sitemap if we particularly want to hide those pages from Google Search. According to Google's documentation, they ignore the priority completely, though I'm not sure if that is really completely correct. However, listing pages with zero priority doesn't feel right. Therefore, `class HiddenBaseExerciseSitemap` should be removed.

> Google ignores `<priority>` and `<changefreq>` values.
https://developers.google.com/search/docs/advanced/sitemaps/build-sitemap#general-guidelines

**There is a better way to hide pages from Google Search.** It requires a small change in the HTML templates to add a new `<meta>` tag. Since this PR is small, I think you could still add it to this same PR. (Or make another PR after this one.) Note that the noindex tag must only affect the exercises that used to be in `HiddenBaseExerciseSitemap`.
> `<meta name="robots" content="noindex">`
https://developers.google.com/search/docs/advanced/crawling/block-indexing?hl=en#meta-tag